src/styles.css

```diff
@@ -28,48 +28,42 @@ footer {
 }
 
 
-/*css of running instances */
+/* Running instances */
 .task-running-on-time > rect {
     stroke-dasharray: 10, 5;
     animation-name: dash-task;
-    animation-duration: 3s;
+    animation-duration: 0.5s;
     animation-timing-function: linear;
     animation-iteration-count: infinite;
```

Comment:
ℹ️ in the 3 CSS rules, everything is duplicated except the filter property.
We could introduce a shared rule and apply it to all elements in addtion to the existing classes.
This would remove the duplication and show how to apply several CSS classes with the API.

Reply:
yes and the animation-duration of course! 😝

Reply:
If we go in that way, let's do it in a dedicated PR

Reply:
👍